### ISSUE TITLE
Bad workspace URL leads to sentry error popup

### ISSUE THREAD

**Issue by blawney** [web-mev/mev-frontend#33]
Some users are directly navigating to a workspace that cannot be found (`https://webmev.tm4.org/#/workspace/<UUID>/`). 

A notification is shown, but the page still generates the Sentry error popup

**Comment by blawney** [comment]
Partially addressed in 8e18487b9693426f4f58478331f75260af2e9878

Now shows N/A on page load.

Need to check behavior when other tabs are loaded.

**Comment by blawney** [comment]
If the user then tries to select other tabs in the workspace detail, then we get problems since each tab has its own API calls which generate 404's. In principle, they should stop after navigating to a dead end, but we can't guarantee that. Clicking around will ultimately lead to errors and the dreaded Sentry popup.

I envision two solutions to this:
- Find all the api calls which can generate 404's and handle that error appropriately (e.g . show empty tables and other messages that indicate there was no data since the workspace UUID was not found)
- Change the angular routing. As it stands, the `WorkspaceDetailComponent` loads based on the routing in https://github.com/web-mev/mev-frontend/blob/main/projects/web-mev/src/app/features/workspace-detail/workspace-detail-routing.module.ts . Even if the workspace UUID is not found, the component still loads, which forces us to ensure that all the API calls are "guarded for failure". Unfortunately, there seems to be no way for Angular to check that the UUID exists in the database first and *then* load the component. One possible workaround is to create a new "container" component which will inspect the URL and call the API to see if the workspace UUID is valid/found. Depending on the results of that, the container component will either load the child `WorkspaceDetailComponent` or an error message. For instance, the container's HTML could be like:

```
<ng-container *ngIf="validWorkspaceUUID">
    <mev-workspace-detail></mev-workspace-detail>   
</ng-container>
<ng-container *ngIf="!validWorkspaceUUID">
    <p>Bad workspace!</p>  
</ng-container>
```

**Comment by blawney** [comment]
Much cleaner solution- just navigate away from bad workspace UUIDs. Fixed in 1b78fe629fffd888de6ffefad73e236bd6b67907
